apps/xprof_gui/priv/src/utils/CommonUtils.js

```diff
@@ -17,34 +17,6 @@ export const commonArrayPrefix = (sortedArray) => {
   return commonPrefix(string1, string2);
 };
 
-export const parseToMfa = fullName => [
-  fullName.split(':')[0],
-  fullName.split('/')[0].split(':')[1],
-  +fullName.split('/')[1],
-  fullName,
-];
-
-export const getLanguageGuides = (mode) => {
```

Comment:
Im lost tracking down how language specific query placeholder gets its value.
So is this function not used for that any more?

Reply:
`getLanguageGuides` function was a remnant from old ui. Now following action creator is used for language detection:
https://github.com/Appliscale/xprof/blob/utils-test/apps/xprof_gui/priv/src/actions/NavigationActions.js#L124

Reply:
gotcha, thanks